pkg/workloadmeta/types.go

```diff
@@ -618,15 +635,14 @@ type Event struct {
 	// When Type is EventTypeUnset, this represents a removed entity.
 	Type EventType
 
-	// Entity is the entity involved in this event.  For a set event, this may
-	// contain information "merged" from multiple sources, as listed in the
-	// Sources field.  For an unset event, it may contain incomplete information,
-	// perhaps only an EntityID.
+	// Entity is the entity involved in this event.  For an EventTypeSet event,
+	// this may contain information "merged" from multiple sources, as listed
+	// in the Sources field.  For an unset event it contains only an EntityID.
```

Comment:
Everything is correct except the `Sources` field is no more.

Reply:
Good catch, thanks!